frontend/justfile

```diff
@@ -25,6 +25,23 @@ export FRONTEND_PNPM_VERSION := `grep 'packageManager' ../package.json | awk -F'
 @pnpm-version:
     echo $FRONTEND_PNPM_VERSION
 
+##########
+# Docker #
+##########
+
+# This section exists because the frontend is not a part of the Docker-based setup
+# and cannot be built easily like `just dc build <service>`.
+
+# Build the frontend Docker image
+build-img tag="openverse-frontend:local" target="app":
```

Comment:
Could this be just `build`, for consistency with the top-level `just build` command (even if we can't incorporate it in `just build` easily)?

Reply:
I kept the name different in case we adopted the Docker model for the frontend. I recalled that someone on the team (maybe @sarayourfriend?) had suggested something along those lines earlier.